lib/taskana-core/src/main/java/pro/taskana/impl/jobs/JobTaskRunner.java

```diff
@@ -0,0 +1,60 @@
+package pro.taskana.impl.jobs;
+
```

Comment:
create JobTasksRunner in package pro.taskana.impl, then TaskanaEngineImpl.openConnection, returnConnection etc don't need to be public.